db/install.xml

```diff
@@ -14,7 +14,7 @@
         <FIELD NAME="contenttype" TYPE="char" LENGTH="100" NOTNULL="true" SEQUENCE="false"/>
         <FIELD NAME="interactions" TYPE="int" LENGTH="10" NOTNULL="true" DEFAULT="0" SEQUENCE="false"/>
         <FIELD NAME="modalcontent" TYPE="text" NOTNULL="false" SEQUENCE="false"/>
-        <FIELD NAME="cohorts" TYPE="text" NOTNULL="false" SEQUENCE="false"/>
+        <FIELD NAME="cohorts" TYPE="char" LENGTH="1333" NOTNULL="true" SEQUENCE="false"/>
```

Comment:
It's the same as the URL field. Otherwise it is arbitrary. It just needs to be large enough to hold any realistic list of cohorts.

Reply:
Thanks for clarifying. I don't see an major concerns with it being a large arbitrary number, probably raises some future questions (from others).

If we needed to do more complicated operations, this would move out to its own table (e.g. foreign key and all that jazz), so for now it should be fine.